Added support for subdir as owner consumption (#4190)
paperless-ngx/paperless-ngx: docs/configuration.md

    Defaults to false.

`PAPERLESS_CONSUMER_SUBDIR_AS_OWNER=<bool>`

: Set the name of the first subdirectory as owner for consumed files. E.g.
`<CONSUMPTION_DIR>/user1/file.pdf` will set the owner with username "user1" to the consumed

> file. Paperless will not create a user that don't exist yet and the file will not have an owner.

Reviewer: ```suggestion
file. Paperless will not create users that do not exist, instead the file will have no owner set.
```

>     This is useful for if you have different users in your system. Each user places
>     their files in the their own folders. These folders won't be deleted.

Reviewer: ```suggestion
```
